quantities/units/Imperial/Mass/Pound.cs

```diff
@@ -1,12 +1,9 @@
 ﻿using Quantities.Dimensions;
-using Quantities.Units.Transformation;
 
 namespace Quantities.Units.Imperial.Mass;
 
 public readonly struct Pound : IImperialUnit, IMass
 {
-    private static readonly Transform transform = new(0.45359237 /* kg */);
-    public static Double ToSi(in Double nonSiValue) => transform.ToSi(in nonSiValue);
-    public static Double FromSi(in Double siValue) => transform.FromSi(in siValue);
+    public static Transformation ToSi(Transformation self) => 45359237 * self / 100000000;
```

Comment:
Why didn't you use the decimal value?

Reply:
Because it's actually more precise and test-ably so! The test "" fails if I use the decimal value with the following message:
```text
[xUnit.net 00:00:01.27]     Quantities.Test.MassTest.GramToPound [FAIL]
  Failed Quantities.Test.MassTest.GramToPound [4 ms]
  Error Message:
   Assert.Equal() Failure
Expected: 3
Actual:   2.9999999999999996
```

It's kinda nice to see that my "obsession" playing around with various options to represent a transformation pays out :-)